config/roles.yml

```diff
@@ -482,6 +482,7 @@ search_relevance_full_access:
         - '*'
       allowed_actions:
         - 'indices:admin/mappings/get'
+        - 'indices:data/read/get*'
         - 'indices:data/read/search*'
```

Comment:
I'd recomment to just use the action group `read` here. That way you can be sure that you have all permissions.